docs/DeveloperGuide.md

```diff
@@ -358,23 +358,23 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **MSS**
 
 1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to add a person not in the list
-4.  AddressBook adds the person
+2.  ConcertPhonebook shows a list of persons
+3.  User requests to add a person
+4.  ConcertPhonebook adds the person
 
     Use case ends.
 
 **Extensions**
 
 - 3a. The list already contains the person's name.
```

Comment:
Outdated duplication criteria here, should be name, phone and email

Reply:
Can either put the duplication criteria or simply put The list already contains the person.